pkgs/development/compilers/llvm/common/bolt/setup-hook.sh.in

```diff
@@ -0,0 +1,16 @@
+# shellcheck shell=bash
+
+function boltFixupPhase {
+  local i
+  while IFS= read -r -d $'\0' i; do
+    if ! isELF "$i"; then continue; fi
+    if ! @readelf@ --program-headers "$i" | grep -q LOAD; then continue; fi
+
+    llvm-bolt "$i" -o $(dirname "$i")/.bolt-$(basename "$i") "${boltFlags[@]-}"
+    mv $(dirname "$i")/.bolt-$(basename "$i") "$i"
+  done < <(find "$prefix" -type f -print0)
+}
+
+if [ -z "${dontUseBolt-}" ]; then
```

Comment:
I wonder if there's a more precise verb than "use" that would work better here…

Reply:
Not sure